### PR TITLE
Tcf day of

### PR DESCRIPTION
**Pull request by wgervasio**
add /tcf/day-of
modify a few typos in aristanetoworks.md, safesoftware.md and activisiondemonware.md
add several diagrams and pdfs to /files/tcf2024/

### CODE REVIEW

**Comment by kewbish** [comment]
Is this ready to be merged?

**Comment by wgervasio** [comment]
nope, will ping when it's good!

**Comment by wgervasio** [comment]
Ready! @kewbish

**Comment by wgervasio** [comment]
most likely broke several style rules in that file

File: themes/hugo-bootstrap-5/layouts/tcf/tcf-day-of.html
Comment (wgervasio) [inline themes/hugo-bootstrap-5/layouts/tcf/tcf-day-of.html:284]: this is a temporary fix. I will add to the company markdowns the true venue numbers next week. Rn is not the time

File: content/tcf/activisiondemonware.md
Comment (kewbish) [inline content/tcf/activisiondemonware.md:6]: Good catch :skull:

**Review by kewbish — APPROVED** [review]
Will approve to unblock, I'll let you merge if you want to make changes re. Youssef's edits!

**Comment by wgervasio** [comment]
merci!

**Pull request merged**
